paper/manuscript.tex

```diff
@@ -48,132 +51,165 @@
-
-The powerful curve-fitting engine offers the choice of using a normal least-squares fitting approach (with a variety of global and gradient-based optimizers) or a Bayesian approach using Markov Chain Monte Carlo to investigate the posterior distribution of the model parameters. The Bayesian approach is useful in examining parameter covariances, model selection, and variability in the resulting scattering length density profiles. Whilst the main intention of the program is to analyse reflectometry data it is equally adept for generalised curve-fitting purposes.
-
-The whole package is designed with a focus on reproducible research; its use in \Jupyter\ notebooks (which includes a graphical user interface), and subsequent distribution of those notebooks as supporting information, permits scientists to straightforwardly reproduce analyses made by others.
+Model construction is modular, being composed from a series of components that each describe a subset of the interface, parameterised in terms of physically relevant parameters (volume fraction of a polymer, lipid area per molecule, etc). The model and data are used to create an objective, which is used to calculate residuals, log-likelihood, and log-prior probabilities of the system. Objectives are combined to perform co-refinement of multiple datasets, and mixed-area models. Prior knowledge of parameter values is encoded as probability distribution functions or bounds on all parameters in the system. Extra prior probability terms can be defined for Components, over and above those available from the Parameters alone. Algebraic Parameter constraints are available.
+A choice of fitting approaches is available include least-squares (global and gradient-based optimizers) and a Bayesian approach using Markov Chain Monte Carlo to investigate the posterior distribution of the model parameters. The Bayesian approach is useful in examining parameter covariances, model selection, and variability in the resulting scattering length density profiles.
```

Comment:
No generalised curvefitting ability?

Reply:
The abstract was about a factor of 2 too long so something had to give...